tests/tests.xml.in

```diff
@@ -1,6 +1,6 @@
 <testdefinition version="0.1">
   <suite name="@PACKAGENAME@-tests" domain="System Software">
-    <set name="@PACKAGENAME@-simple_tests" description="basic iodata functionality" feature="System Control" requirement="MaSSW-815, MaSSW-1167">
+    <set name="@PACKAGENAME@-simple_tests" description="basic iodata functionality" feature="Libiodata System Control" requirement="MaSSW-815, MaSSW-1167">
```

Comment:
That "System control" part possibly somehow comes from: "libiodata" is effectively part of "timed", which in olden times used to belong to system control team -> might as well drop it altogether and use something like feature="Libiodata time daemon data parsing" or stte. But does not really matter & just fine as it is.

Reply:
Maybe i'll just call that libiodata now?